dwave_networkx/drawing/chimera_layout.py

```diff
@@ -51,6 +51,11 @@ def chimera_layout(G, scale=1., center=None, dim=2):
         Number of dimensions. When dim > 2, all extra dimensions are
         set to 0.
 
+    normalize_kwargs : None or dict (default None)
+        A dict of keyword arguments to be used in a plotting function.  If not
+        None, we will populate the "ax" keyword with matplotlib axes, and the
+        "node_size" and "width" keywords with defaults if they are not set.
+
```

Comment:
```
    normalize_kwargs : None or dict (default None)
        A dict of keyword arguments to be used in a plotting function (see
        :func:`draw_chimera`, :func:`draw_chimera_embedding` or
        :func:`draw_yield`).  Populates the "ax" keyword with matplotlib axes,
        and the "node_size" and "width" keywords with defaults if they are not
        set.  If normalize_kwargs is None, do nothing.
```

Tense mismatch?

Reply:
Shouldn't the link be to :func:`draw_lineplot`, that is where I found the params I could play with? 
Please use double back quotes (``) for params instead of double quotes (")

Reply:
What happens is `normalize_kwargs["ax"]` gets set to an axis object -- it's an awkward case because it's both a keyword and a string.  Still think I should use double-backquotes?

Reply:
I like this better, but I'll apply your advice to `normalize_size_and_aspect`
```
    normalize_kwargs : None or dict (default None)
        A dict of keyword arguments to be used in a plotting function (see
        :func:`networkx.draw` and :func:`draw_lineplot`).  Populates several
        keys in ``normalize_kwargs`` by calling :func:`normalize_size_and_aspect`
        with appropriate parameters for G. If normalize_kwargs is None, do 
        nothing.
```

Reply:
"Populates several keys in ``normalize_kwargs``" is a bit off: the dict isn't populated here, it can be set here by the user, and if used, the called function sets default values for others. So probably be clearer if the 2nd sentence said something like "Optional keywords that are not provided are set to default values by the :func:`normalize_size_and_aspect` function" 

BTW, is "normalize" clearer in this context than something like "matplot_kwargs" or "graph_kwargs" or "plot_kwargs"? If I just saw the keyword I would not connect it to making the graphing eye friendly

Reply:
Eh, "plot_kwargs" is fewer characters.  Yeah, I hated that name.

Reply:
```
    plot_kwargs : None or dict (default None)
        A dict of keyword arguments to be used in a plotting function (see
        :func:`networkx.draw` and :func:`draw_lineplot`), to scale node and edge
        sizes appropriately to the graph `G`.  Optional keys in ``plot_kwargs``
        are set to default values by calling :func:`normalize_size_and_aspect`
        with appropriate parameters for G. Do nothing if ``plot_kwargs`` is None
```

Reply:
You could also mention that you added coloring capability. Maybe something like

plot_kwargs : None or dict (default None)
        Parameters to shape and color plotting, as a dict of keyword arguments (see
        :func:`networkx.draw` and :func:`draw_lineplot`).  Unspecified parameters of
        ``plot_kwargs`` are set to default values appropriate to the graph by the underlying
        :func:`normalize_size_and_aspect` call. Does nothing if ``plot_kwargs`` is None.

Reply:
I see now that I have not documented `normalize_size_and_aspect`.  I don't do anything with coloring there -- what's getting normalized is
 1. if an axis (`"ax"`) is not set, then I create one
 2. the aspect ratio of the axis is set to 1
 3. axes are turned off (this is confusing because of the overloaded term "axis" -- here I mean that we will not draw lines and tick marks for the x- and y-axes on the plot)
 4. if `"node_size"` and/or `"width"` are not set, they are calculated from the fabric scale, figure scale, and magic numbers

So, in steps (1-3), the matplotlib `"axis"` object is being normalized.  In step 4, the feature sizes in the drawings are being normalized.